### PR TITLE
Issue #13345: Enable examples tests for RegexSinglelinejavaCheck

### PR DESCRIPTION
**Pull request by AmitKumarDeoghoria**
### Title: Enable examples tests for RegexSinglelinejavaCheck
**Related Issue: #13345** 

**Summary**
This pull request aims to enable example tests for the **  RegexSinglelinejavaCheck** in the project. The implementation focuses on ensuring that   RegexSinglelinejava checks are correctly validated through comprehensive test cases, improving overall code reliability and quality.

**Key Changes Made**

- Conversion of txt file to Java File
- Enable the test cases by adding expected violations in the expected array for all example tests
- Made corresponding changes in the xml and xml.template file

### CODE REVIEW

**Comment by AmitKumarDeoghoria** [comment]
GitHub, generate website

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example3.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RegexpSinglelineJava">
+      <property name="format" value="Example3"/>
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example3.java:5]: Please make value to be "debug".
And rename some variable in code to be this name

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example2.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    out.write("something");
+  }
+
+  public void testMethod3(){
+    final Logger logger = Logger.getLogger(Example2.class.getName());
+
+    logger.info("first");
+    logger.info("second");
+    logger.info("third");
+    System.out.println("fourth"); // violation, instruction matches illegal pattern
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example2.java:48]: Please use message of violation. There were few more above

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example3.java (hunk 1 above)
Comment (AmitKumarDeoghoria) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example3.java]: Done

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example2.java (hunk 2 above)
Comment (AmitKumarDeoghoria) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsinglelinejava/Example2.java]: Done

**Review by romani — APPROVED** [review]

**Pull request merged**